picard/ui/mainwindow.py

```diff
@@ -903,31 +904,39 @@ def _update_cd_lookup_actions(self, result=None, error=None):
             self.update_cd_lookup_drives(result)
 
     def update_cd_lookup_drives(self, drives):
+        self.cd_lookup_menu.clear()
+        self.cd_lookup_action.setEnabled(discid is not None)
         if not drives:
             log.warning("CDROM: No CD-ROM drives found - Lookup CD functionality disabled")
+            has_drives = False
         else:
+            has_drives = True
```

Comment:
I don't think we need `has_drives`, since we can use `drives` or `bool(drives)`.

Reply:
True, I changed it to just pass `drives`  around